src/consume.ts

```diff
@@ -656,18 +658,17 @@ function messageViewerStartPollingCommand(
           serialized.value.includes(index),
         );
 
-        // i want to drop the comment in favor of filename and possibly do a preview tab
-        workspace
-          .openTextDocument({
-            content: `// message ${message.key} from ${topic.name}\n${JSON.stringify(payload, null, 2)}`,
-            language: "jsonc",
+        // read-only document buffer with the message content
+        const filename = `${topic.name}-message-${index}.json`;
```

Comment:
Specifically not using `key` here since it could include a JSON blob or longer data.